tests/integration/helpers/test_tools.py

```diff
@@ -25,7 +27,7 @@ def __init__(self, contents):
                 "contents must be either file or string or list, actual type: "
                 + type(contents).__name__
             )
-        self.lines = [l.strip() for l in raw_lines if l.strip()]
+        self.lines = [content.strip() for content in raw_lines if content.strip()]
```

Comment:
What's wrong with `l`?

Reply:
Easy to confuse `l` with `1`.

% `ruff rule E741`
# ambiguous-variable-name (E741)

Derived from the **pycodestyle** linter.

## What it does
Checks for the use of the characters 'l', 'O', or 'I' as variable names.

## Why is this bad?
In some fonts, these characters are indistinguishable from the
numerals one and zero. When tempted to use 'l', use 'L' instead.

## Example
```python
l = 0
O = 123
I = 42
```

Use instead:
```python
L = 0
o = 123
i = 42
```